eslint.config.mjs

```diff
@@ -7,6 +7,17 @@ export default tseslint.config(
     eslint.configs.recommended,
     ...tseslint.configs.recommended,
     {
+        languageOptions: {
+            // Add node globals to ignore undefined
+            globals: {
+                "__dirname": false,
+                "console": false,
+                "module": false,
+                "process": false,
+                "require": false,
+                "setTimeout": false,
+            }
+        },
```

Comment:
An alternative to this would be:
- Install the `globals` npm package: https://www.npmjs.com/package/globals
- Import those predefined globals into the config file, e.g.

```mjs
globals: {
  ...globals.node,
}
```

I like defining them as we go, so we have one less dependency, but if this becomes unmanageable, then we can switch to predefined globals.

Reply:
I agree on having one less dependency unless the management side outweighs the benefit of less dependencies. Let's leave it as is until it really becomes a problem :)